ftdc/ftdc.go

```diff
@@ -536,7 +553,7 @@ func (ftdc *FTDC) checkAndDeleteOldFiles() error {
 // deletion testing. Filename generation uses padding such that we can rely on there before 2/4
 // digits for every numeric value.
 //
-//nolint
+// nolint
```

Comment:
It joys me that our linter lints our linter

Reply:
Or maybe this was gofmt. The linter complained and it's now back to its unindented state.